src/main/java/org/opensearch/security/ssl/util/CertFileProps.java

```diff
@@ -1,3 +1,14 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
should we add license headers to all un-licensed files?

Reply:
I've done the minimum work required for the license check from OpenSearch's build system to pass.  I would recommend we create another issue if we want to refine the scope/settings of that tool

Reply:
okay... if this is required by the core's build system to pass then we do not need to create a separate issue to discuss on this.